pkg/linter/linter.go

```diff
@@ -369,7 +374,7 @@ func (l Linter) paramsToRulesConfig() map[string]any {
 	}
 }
 
-func (l Linter) prepareRegoArgs() []func(*rego.Rego) {
+func (l Linter) prepareRegoArgs(query ast.Body) []func(*rego.Rego) {
```

Comment:
This looks unused?

Reply:
It's used by `lintWithRegoRules` (where query is `violations = data.regal.main.report`) and by `collectAggregates` (where query is `aggregates = data.regal.main.aggregate`)

Reply:
Ah, all good!